component/pom.xml

```diff
@@ -20,7 +20,7 @@
     <parent>
         <groupId>org.ballerinalang</groupId>
         <artifactId>connector-mongodb-parent</artifactId>
-        <version>0.8.3-SNAPSHOT</version>
+        <version>0.8.4-SNAPSHOT</version>
```

Comment:
Why have we updated the snapshot version here?

Reply:
I have updated this version since we need to release with the new changes. Don't we need to do this?

Reply:
When we are releasing, next snapshot version is automatically updated by Maven release plugin therefore we don't have to manually update this.